docs/sources/guides/whats-new-in-v5-1.md

```diff
@@ -0,0 +1,111 @@
+
+Version | User    | User ID
+--------|---------|---------
+< 5.1   | grafana | 104
+>= 5.1  | grafana | 472
+
+Please read the [updated documentation](/installation/docker/#migration-from-a-previous-version-of-the-docker-container-to-5-1-or-later) which includes migration instructions and more information.
+
+## Improved workflow for provisioned dashboards
+
+{{< docs-imagebox img="/img/docs/v51/provisioning_cannot_save_dashboard.png" max-width="800px" class="docs-image--right" >}}
+
+Grafana v5.1 brings an improved workflow for provisioned dashboards:
+
+* A populated `id` property in JSON is now automatically removed when provisioning dashboards.
+* When making changes to a provisioned dashboard you can `Save` the dashboard which now will bring up a *Cannot save provisioned dashboard* dialog like seen in the screenshot to the right.
+
+
+Available options in the dialog will let you `Copy JSON to Clipboard` and/or `Save JSON to file` which can help you synchronize your dashboard changes back to the provisioning source.
+More information in the [Provisioning documentation](/features/datasources/prometheus/).
+
+<div class="clearfix"></div>
+
+## New template variable interpolation syntax
+
+
+## Alerting
+
+New enhancements includes support for retries on failed alert queries and updating correct state when pausing/unpausing alerts.
+
+## Datasources
+
+### Prometheus
+
+New enhancements includes support for Prometheus histograms in the heatmap panel and autocomplete of template variables query editor. More information in the [Prometheus data source documentation](/features/datasources/prometheus/).
+
+### Cloudwatch
+
+New enhancements includes support for high resolution metrics and dimension filtering. More information in the [Cloudwatch data source documentation](/features/datasources/cloudwatch/).
+
+### PostgreSQL
+
+New enhancement includes support for filling in values for missing intervals and better precision handling and data type support for time columns. More information in the [PostgreSQL data source documentation](/features/datasources/postgres/#time-series-queries).
+
+### MySQL
+
+New enhancements includes support for filling in values for missing intervals, better precision handling and data type support for time columns and any column except time and metric is now treated as a value column. More information in the [MySQL data source documentation](/features/datasources/mysql/#time-series-queries).
+
+### Microsoft SQL Server
+
+{{< docs-imagebox img="/img/docs/v51/mssql_query_editor_showcase.png"  max-width= "800px" class="docs-image--right" >}}
+
+Grafana v5.1 now ships with a built-in Microsoft SQL Server (MSSQL) data source plugin that allows you to query and visualize data from any Microsoft SQL Server 2005 or newer, including Microsoft Azure SQL Database. Have logs or metric data in MSSQL? You can now visualize that data and
+define alert rules on it like any of our other data sources.
```

Comment:
Isn't it better to write it from the user perspective and say `you other data sources` instead of `our other data sources`?

Reply:
Good point. However, we only supports alerting for the core datasources in Grafana so may be a bit weird to write "your". Will try to come up with something else.